File: C++/program132.cpp
#include<iostream>
using namespace std;

bool Search(int Arr[], int iSize)
{
    int iCnt = 0;
    bool bFlag = false;

    for(iCnt = 0; iCnt < iSize; iCnt++)
    {
        if((Arr[iCnt] ==  5))
        {
            bFlag = true;
        }
    }

    return bFlag;
}

int main()
{
    int iLength = 0, iCnt = 0;
    bool bRet = false;
    int *ptr = NULL;

    cout<<"Enter number of elements "<<'\n';
    cin>>iLength;

    ptr = new int[iLength];

    if(NULL == ptr)
    {
        cout<<"Unable to allocate memory"<<'\n';
        return -1;
    }

    cout<<"Enter array elements "<<'\n';
    for(iCnt = 0; iCnt < iLength; iCnt++)
    {
        cin>>ptr[iCnt];
    }

    bRet = Search(ptr,iLength);
    
    if(bRet == true)
    {
        cout<<"Element found"<<'\n';
    }
    else
    {
        cout<<"Element not found"<<'\n';
    }

    delete [] ptr;
    return 0;
}

File: Java/program132.java
import java.util.*;
class ArrayProgram
{
    public boolean Search(int [] Arr, int iSize)
    {
        int iCnt = 0;
        System.out.println("Entered  elements are :");

        for(iCnt = 0; iCnt < iSize ; iCnt++)
        {
           if(Arr[iCnt] == 11)
           {
                break;
           }
        }

        if(iCnt == iSize)
           {
                return false;  
           }
           else
           {
                return true;
           }

        

    }
}

class program132
{
    public static void main(String A[])
    {
        int iCnt = 0, iLength = 0 ;
        boolean bRet = false;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of elements ");
        iLength = sobj.nextInt();

        int [] Arr = new int [iLength];

        System.out.println("Enter elements :");

        for(iCnt = 0; iCnt < iLength ; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt() ;
        }

       

        ArrayProgram aobj = new ArrayProgram();
        bRet = aobj.Search(Arr,iLength);

        if(bRet == true)
        {
             System.out.println("Element is Found ");
        }
        else
        {
            System.out.println(" Element is  not Found ");
        }
       

        

    }
}